frontend/SideDish/SideDish/ViewModel/MenusViewModel.swift

```diff
@@ -50,27 +53,67 @@ class MenusViewModel {
         }
     }
     
-    func matchingViewModel(menuList: [Menu]) -> [MenuViewModel] {
+    private func matchingViewModel(menuList: [Menu]) -> [MenuViewModel] {
+        
         let viewModelList: [MenuViewModel] = menuList.map() { menu in
-            let nPrice = stringToAttributedString(menu.nPrice)
-            let viewModel = MenuViewModel(image: menu.image, title: menu.title, body: menu.description, sPrice: menu.sPrice, nPrice: nPrice, badges: menu.badge ?? [])
+            let price = menu.normalPrice == nil ? nil : String(menu.normalPrice!)
```

Comment:
강제 언래핑은 가급적 사용 자제하세요. 값이 잘못 들어온다면 앱 크래시가 발생합니다.

```suggestion
var price: String?          
if let normalPrice = menu.normalPrice {
    price = String(normalPrice)
```